server.go

```diff
@@ -396,10 +408,24 @@ func (s *Server) processStreamingRPC(t transport.ServerTransport, stream *transp
 			ss.statusDesc = appErr.Error()
 		}
 	}
+	if ss.tracing {
+		ss.mu.Lock()
+		if ss.statusCode != codes.OK {
+			ss.traceInfo.tr.LazyLog(stringer(ss.statusDesc), true)
+			ss.traceInfo.tr.SetError()
+		} else {
+			ss.traceInfo.tr.LazyLog(stringer("OK"), true)
```

Comment:
LazyPrintf does exactly what LazyLog+fmtStringer does, except it doesn't have a control for sensitivity. There is indeed a struct overhead, though I suspect it would be negligible. Not too fussed.